### ISSUE TITLE
[DONATION] ShortIntervalTask

### ISSUE THREAD

**Issue by AidenRushbrooke** [uea-machine-learning/tsml_repo#104]
https://pubmed.ncbi.nlm.nih.gov/37263767/
https://openneuro.org/datasets/ds004579/versions/1.0.0
This dataset aims to look at the impact of Parkinsons disease on the completion of a basic task. 94 participants with Parkinsons, and 45
control participants were asked to press down the spacebar on a keyboard for what they judge to be 3 seconds. Each participant completed task 20 times, with the EEG data being recorded at 500Hz with 63 channels. Each instance is created from a 5.5 second window around each button press, starting 0.5 seconds before the inital prompt was shown to the participant. For the train-test split,
the first 44 participants with Parkinsons, and the first 22 control were used for the training set, and the remaining formed the test set.
